### PR TITLE
Update cveCreateSession.md

### PR DESCRIPTION
**Pull request by hparkersd**
Updated to reflect changes on login screen and included new features associated with the New Applicant notification send to SMs. Minor updates.

### CODE REVIEW

**Review by saitohirga — APPROVED** [review]
Looks good here.

File: content/docs/cve/cveCreateSession.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ type: book  # Do not modify.
 
 Detailed instructions to schedule an exam session in the ExamTools software are written below.  You may also watch [this video](https://www.youtube.com/watch?v=vyoIsFpNkZ8&list=PLDxsQ4bjUNnppBmsjEE9vad-ZuQonD8Bp&index=3) for a walkthrough.
 
-* Login to **[ExamTools](http://Exam.Tools)** by clicking on [SIGN IN] under Assist with an Exam.
+* Login to **[ExamTools](http://Exam.Tools)** by clicking on [SIGN IN] under Assist with or Manage Exams.
```
Comment (n2ygk) [inline content/docs/cve/cveCreateSession.md:21]: I suggest replacing `http` with `https` wherever it appears.

File: content/docs/cve/cveCreateSession.md [hunk 2]
```diff
@@ -46,29 +46,32 @@ Detailed instructions to schedule an exam session in the ExamTools software are
...
+  * Do not list on HamStudy.org – Hide your session and do not display on HamStudy.  Especially helpful, if you want to work on your session listing ahead of time and aren’t quite ready for applicants or want to share a private link to a certain population for whom the session has been scheduled.  (i.e. Local radio club or college course)  The session is still accessible online, but the applicant will need a private link to view.  When you are ready to list your session on HamStudy, return to the session in ET and uncheck the box and it will appear in the HamStudy list of [Upcoming test sessions](https://ham.study/sessions).
```
Comment (n2ygk) [inline content/docs/cve/cveCreateSession.md:63]: > scheduled. (i.e. Local radio club or college course)
Move the period after `scheduled` to after the closing parenthesis.

File: content/docs/cve/cveCreateSession.md [hunk 3]
```diff
@@ -46,29 +46,32 @@ Detailed instructions to schedule an exam session in the ExamTools software are
...
 
   * **Schedule only** - Your exam will not appear on the schedule and online registration will be disabled.
 
-  You may want to do this to see the posting online, but for some reason do not want the general public to view it.  You may do this to proof-read it, experiment with formatting or share it with your VE team before it goes live.
+  Select the Schedule only and Do not list on HamStudy.org combo when you do not want the public to view it and register.  You may do this to proof-read it, experiment with formatting or share it with your VE team before it goes live using the private link.
+  
+  * **Contact Name and Email** - This field will include the Session Manager's email address by default, but may be changed to a preferred email address as long as the formatting is maintained.  ET uses this email for notification and does not appear in your session listing online.  If you want your email to be posted, added it to the Other Notes below.
```
Comment (n2ygk) [inline content/docs/cve/cveCreateSession.md:72]: > ET uses this email for notification and does not appear in your session listing online.

ET uses this email for notification and it does not appear in your session listing online.

(added `it`)

**Review by nkbooth — APPROVED** [review]
Looks good. Thanks Heather!

File: content/docs/cve/cveCreateSession.md [hunk 4]
```diff
@@ -80,7 +83,7 @@ Detailed instructions to schedule an exam session in the ExamTools software are
...
 
-* Search for your session by clicking on the **Find a Session** from the right side navigation on HamStudy.  If the session is not hidden, then it should come up.  Find all your sessions by adding your callsign to the end of the following link: https://ham.study/sessions/yourcallsign.  Use this url to link to all your sessions live on HamStudy.  Include it in the session descriptions or share in email. This link will not include hidden sessions.
+* Search for your session by clicking on the **Find a Session** from the right side navigation on [HamStudy](https://ham.study).  If the session is not hidden, then it should come up.  Find all your sessions by adding your callsign to the end of the following link: https://ham.study/sessions/yourcallsign.  Use this URL to link to all your sessions live on HamStudy.  Include it in the session description or share in email. This link will not include hidden sessions.
```
Comment (n2ygk) [inline content/docs/cve/cveCreateSession.md:86]: > link: https://ham.study/sessions/yourcallsign.

Maybe format this with back-tics so it doesn't become clickable... Also, if says "add to the end of..." so perhaps leave out `yourcsallsign` or change the wording to say "replace `yourcallsign` with your callsign" or something along those lines.

**Review by n2ygk — CHANGES_REQUESTED** [review]
Just a few grammar nazi changes.

**Comment by n2ygk** [comment]
Hey! I was reviewing here!

**Comment by hparkersd** [comment]
Allan - I will go back and make some of those changes which you suggested.

**Pull request merged**